### DISCUSSION TITLE
Задать дату отправки

### DISCUSSION THREAD

**Discussion by tigerxml** [cdek-it/widget#78] [Ideas]
Добрый день!

Как передать дату отгрузки заказа в ПВЗ? Т.к. не всегда заказ можно отгрузить в день, иногда отгрузка занимает 2 -3 и иногда больше дней.

Нужно иметь возможность чтобы эти дни добавлялись к сроку доставки, который рассчитывается в виджете.
Как это возможно сделать?

**Comment by vermorag** [comment]
Добрый день. Такого функционала в настоящий момент не существует в виджете.
Он будет добавлен с версии 4.0.
